app/views/users/perform_account_activation.html.erb

```diff
@@ -1,22 +1,27 @@
 <div id='simple-container'>
-
   <%if flash[:error]%>
     <div class='alert alert-error'>
       <%=flash[:error]%>
     </div>
   <%else%>
-    <h2><%= t 'activation.welcome_to', site_name: SiteSetting.title %></h2>
-    <br>
-
-    <% if @needs_approval %>
-      <p><%= t 'activation.approval_required' %></p>
-    <% else %>
-      <p><%= t('activation.please_continue') %></p>
-      <p><a class="btn" href="<%= path "/" %>"><%= t('activation.continue_button', site_name: SiteSetting.title) -%></a></p>
-      <%- content_for(:no_ember_head) do %>
-        <%= tag.meta id: 'data-auto-redirect', data: { path: path('/') } %>
-      <%- end %>
-      <%= preload_script 'auto-redirect' %>
-    <% end %>
+    <div class='activate-account'>
+      <h1 class="activate-title"><%= t 'activation.welcome_to', site_name: SiteSetting.title %> <img src="/images/wave.png" alt="" class="waving-hand"></h1>
```

Comment:
Too bad we can't our wavingHand logic here (and have to add the image file…)

Maybe we should convert it to an ember page. I don't think this needs to be rails-only? (cc: @davidtaylorhq)

Reply:
> Maybe we should convert it to an ember page. I don't think this needs to be rails-only?

Yeah that would be great - we should do that. But I don't think it needs to block this redesign.

> our wavingHand logic here (and have to add the image file…)

Our wavinghand logic essentially provides a link to [/images/emoji/twitter/wave.png](https://meta.discourse.org/images/emoji/twitter/wave.png?v=12). We could do that on the server as well. @MeghnaAJ how large is the emoji being rendered? Would 72x72px be good enough?

Reply:
According to the CSS it's 32px, so 72px should be more than enough.

So I think this should work

```suggestion
      <h1 class="activate-title"><%= t 'activation.welcome_to', site_name: SiteSetting.title %> <img src="<%= Emoji.url_for("wave") %>" alt="" class="waving-hand"></h1>
```

And then we don't need to add the extra PNG file.

Reply:
I've made the suggested change. Thanks! 👍